service/logger.go

```diff
@@ -31,26 +31,23 @@ const (
 
 var (
 	// Command line pointer to logger level flag configuration.
-	loggerLevelPtr   *string
+	loggerLevelPtr   *zapcore.Level
 	loggerProfilePtr *string
 	loggerFormatPtr  *string
 )
 
 func loggerFlags(flags *flag.FlagSet) {
-	loggerLevelPtr = flags.String(logLevelCfg, "INFO", "Output level of logs (DEBUG, INFO, WARN, ERROR, DPANIC, PANIC, FATAL)")
+	defaultLevel := zapcore.InfoLevel
+	loggerLevelPtr = &defaultLevel
+	flags.Var(loggerLevelPtr, logLevelCfg, "Output level of logs (DEBUG, INFO, WARN, ERROR, DPANIC, PANIC, FATAL)")
```

Comment:
This makes it looks like you need to pass "DEBUG", "INFO", etc as the flag value.